templates/shared/forms/interactive/managed-kafka.html

```diff
@@ -0,0 +1,205 @@
+<div class="p-modal" id="contact-modal">
+  <div class="p-modal__dialog" role="dialog" aria-labelledby="modal-title" aria-describedby="modal-description">
+    <header class="p-modal__header" style="display: block; border-bottom: 0;">
+      <button class="p-modal__close" aria-label="Close active modal" style="margin-left: -1rem">Close</button>
+      <hr />
```

Comment:
Can you remove this `hr` from all interactive forms and the creation script, please.

Reply:
yes - all as in go fix the other ones?